docs/user-guide/examples/setup-bootstrap.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+kind create cluster --config kind.yaml
+
+# (Optional) Initialize CAPM3. This is only needed for scenario 2, but it also installs
+# cert-manager, which is needed for pretty much everything else.
+# If you skip this, make sure you install cert-manager separately!
+clusterctl init --infrastructure=metal3 --ipam=metal3
+
+kubectl apply -f https://github.com/metal3-io/ironic-standalone-operator/releases/latest/download/install.yaml
+kubectl -n ironic-standalone-operator-system wait --for=condition=Available deploy/ironic-standalone-operator-controller-manager
+
+# Now we can deploy Ironic and BMO
+kubectl create ns baremetal-operator-system
+# Apply Ironic with retry logic (up to 5 attempts with 10 second delays).
+# The IrSO webhoook is not guaranteed to be ready when the IrSO deployment is,
+# so some retries may be needed.
+MAX_RETRIES=5
+RETRY_DELAY=10
+RETRY_COUNT=0
+echo "Applying Ironic configuration..."
+while [ $RETRY_COUNT -lt $MAX_RETRIES ]; do
+  if kubectl apply -k ironic; then
+    echo "Successfully applied Ironic configuration"
+    break
+  else
+    RETRY_COUNT=$((RETRY_COUNT + 1))
+    echo "Failed to apply Ironic configuration. Retrying in ${RETRY_DELAY} seconds... (Attempt $RETRY_COUNT/$MAX_RETRIES)"
+    sleep $RETRY_DELAY
+  fi
+done
```

Comment:
I used this for easy testing. Not sure if we should actually include the retry in the guide or just say "wait for IrSO, then deploy Ironic". What do you think?

Reply:
Fine for me as is.